data/compatible_products.yml

```diff
@@ -27,10 +27,10 @@ sets:
 
     - name: Open Liberty
       vendor: IBM Corporation
-      version: 19.0.0.6
+      version: 20.0.0.3
```

Comment:
I thought we were going to list multiple compatible versions...  This was discussed at a previous Spec Committee call...

Reply:
@kwsutter I was informed that it was decided that we would show the most recent product release level on the compatible products page and leave it up to the vendor to list all versions.